### PR TITLE
Update camel-k-runtime to 1.13.0 in defaults

### PR DESCRIPTION
**Pull request by claudio4j**
Also update the dataformat name coming from camel-k catalog,
the json prefix was removed

**Release Note**
```release-note
NONE
```

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

**Comment by claudio4j** [comment]
Many CI tests failed because it tried to download from https://repo1.maven.org/maven2/ public repo, but as 1.13.0 is not in public repo yet, the package_artifacts.sh should download from staging repo.

File: pkg/util/source/inspector_yaml.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ func (i YAMLInspector) parseStep(key string, content interface{}, meta *Metadata
 				dataFormatID := defaultJSONDataFormat
 				if jsContent, jsContentOk := js.(map[interface{}]interface{}); jsContentOk {
 					if lib, libOk := jsContent["library"]; libOk {
-						dataFormatID = strings.ToLower(fmt.Sprintf("json-%s", lib))
+						dataFormatID = strings.ToLower(fmt.Sprint(lib))
```
Comment (squakez) [inline pkg/util/source/inspector_yaml.go:76]: Should'nt be `fmt.Sprintf("%s", lib)`?

**Review by squakez — APPROVED** [review]

File: pkg/util/source/inspector_yaml.go (hunk 1 above)
Comment (claudio4j) [reply pkg/util/source/inspector_yaml.go]: It's safer to use the string formatter, thanks for noticing.

**Comment by claudio4j** [comment]
The use of staging maven repo allowed to download the camel-k-runtime source zip, but the `mvn dependency:copy-dependencies` failed for the same reason as the camel-k-runtime artifacts could not be resolved.

**Comment by oscerd** [comment]
I think it"s probably safe to try to release and see. Setting up the ci with staging repo is too much headache

**Comment by squakez** [comment]
Agree with @oscerd. After all, the `package-artifact` is only used in the `make images-dev` whose original goal is to help developers to bundle local (SNAPSHOT) dependencies.

**Pull request closed without merging**